libr/core/cmd_print.c

```diff
@@ -2583,22 +2583,22 @@ static int cmd_print(void *data, const char *input) {
 					ut8 *buf;
 					r_cons_printf ("{");
 					r_cons_printf ("\"name\":\"%s\"", f->name);
-					r_cons_printf (",\"size\":%d", f->size);
+					r_cons_printf (",\"size\":%d", r_anal_fcn_size (f));
 					r_cons_printf (",\"addr\":%"PFMT64d, f->addr);
 					r_cons_printf (",\"ops\":");
 					// instructions are all outputted as a json list
-					buf = malloc (f->size);
+					buf = malloc (r_anal_fcn_size (f));
```

Comment:
this is wrong. you cant read the whole function by reading linearly the sum(bbsize)